test.sh

```diff
@@ -0,0 +1,316 @@
+#!/bin/bash
+
+# Function to wait for containers to be up
```

Comment:
Might be good to add a comment at the top that explains generally what `test.sh` does.